indico/core/signals/event/registration.py

```diff
@@ -66,3 +66,8 @@
 If this signal returns ``True``, the user will not be able to download
 their ticket.
 """)
+
+registration_checkin_updated = _signals.signal('registration-checkin-updated', """
+Called when the checkin state of registration changes. The `sender` is the
```

Comment:
> of *a* registration

(same typo in `registration_state_updated`, feel free to fix it there too).

I'd also move your new signal below that one since they are kind of similar so I think it makes more sense to have it up there.

Reply:
Done